org.altaqwaa.Altaqwaa.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.altaqwaa.Altaqwaa
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+separate-locales: false
+rename-icon: org.altaqwaa.rn0x
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: altaqwaa
+    buildsystem: simple
+    build-commands:
+      - mv * /app/bin
+      - mkdir -p /app/share/icons/hicolor
+      - mkdir -p /app/share/applications
+      - mv /app/bin/hicolor /app/share/icons/
+      - mv /app/bin/${FLATPAK_ID}.desktop /app/share/applications/
+      - install -Dm644 -t=/app/share/metainfo /app/bin/${FLATPAK_ID}.appdata.xml
+    sources:
+      - type: archive
+        url: https://github.com/rn0x/Altaqwaa-Islamic-Desktop-Application/releases/download/v3.0.1/altaqwaa-3.0.1.tar.gz
+        sha256: ed7bd6ce3876e95a973f06247b0d1ed41858dbae46958f48df0d8bb2ad00ee7b
+      # Wrapper to launch the app
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - zypak-wrapper.sh /app/bin/altaqwaa "$@"
+      - type: file
+        path: org.altaqwaa.Altaqwaa.desktop
+      - type: dir
+        path: icons
```

Comment:
1. please don't use `dir`
2. you don't need the small icons. Just put the 256x256. Ideall they should be managed upstream

Reply:
The builds don't have any sort of desktop integration or icons, so I'll have the 256x256 for now. I'll also ask the author if they can make a scalable svg one instead.